README.md

```diff
@@ -6,6 +6,46 @@
 # TheGame
 A basic 2D game with no current ideas for story, setting, etc.
 
+# Setting Up a Virtual Environment
+Before setting up the virtual environment, we first need to install virtualenv,
+```commandline
+python -m pip install virtualenv
+```
+
+then you need to create a virtual environment. First navigate to the
+directory that you would like to create the virtual environment at.
+ It is recommended *not* to create this in the thegame/ directory as
+ this may cause some issues if running black .
+```commandline
+python -m virtualenv venv
+```
+
+### Windows
+In order to activate the virtual environment that was created, run
+```commandline
+venv\Scripts\activate.bat
+```
+
+Once this has been run, before your pwd, ```(thegameenv)``` should appear before it.
```

Comment:
The name (thegameenv) would not match with the other commands. Based on the commands, it would be (venv).

Reply:
Ahh, great catch thank you, i was calling it that previously, but didnt want to update the .gitignore, will fix

Reply:
Addressed with https://github.com/Ross-Alexandra/TheGame/pull/24/commits/8f2c986fcf0e0ae365387424ca5dd4964dac5533